src/patternfly/components/Page/page.scss

```diff
@@ -34,20 +34,27 @@
 
   // Header nav
   --pf-c-page__header-nav--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-page__header-nav--md--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-page__header-nav--md--PaddingLeft: var(--pf-global--spacer--lg);
   --pf-c-page__header-nav--lg--PaddingLeft: 0;
   --pf-c-page__header-nav--lg--MarginLeft: var(--pf-global--spacer--xl);
   --pf-c-page__header-nav--lg--MarginRight: var(--pf-global--spacer--xl);
-  --pf-c-page__header-nav--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-100);
+  --pf-c-page__header-nav--BackgroundColor: #242424; // Custom color for this element only
   --pf-c-page__header-nav--lg--BackgroundColor: transparent;
+  --pf-c-page__header-nav--c-nav__scroll-button--nth-of-type-1--Left: calc(-1 * (var(--pf-global--spacer--md) - var(--pf-global--spacer--xs))); // This sets the the button to the edge of its container with 4px clearance on the left
+  --pf-c-page__header-nav--c-nav__scroll-button--nth-of-type-1--md--Left: calc(-1 * (var(--pf-global--spacer--md) - var(--pf-global--spacer--xs)));
```

Comment:
Should we also do this to the 2nd button to make padding balanced on both sides

<img width="854" alt="Screen Shot 2019-05-13 at 9 56 26 AM" src="https://user-images.githubusercontent.com/20118816/57627185-7fee9680-7565-11e9-812c-0ea2442d3922.png">

Reply:
To do that, if we move the button to the left a little, I think we'll need to add right padding to the horizontal nav so that the nav links don't show on the right side of the button (behind/underneath the button, since the nav extends to the right edge of the page). That would probably be a breaking change for folks who don't have the overflow buttons enabled, since their nav would no longer cut off/overflow to the edge of the browser and would cut off at the padding instead. But if we're OK with the way this looks for folks that don't have the overflow buttons enabled, I think that's a good change.

What do you think @kybaker, is this OK?

<img width="860" alt="Screen Shot 2019-05-17 at 9 28 13 AM" src="https://user-images.githubusercontent.com/35148959/57934933-44660c00-7886-11e9-8679-57a179c0bda8.png">

FWIW, this is what it looks like currently.

<img width="854" alt="Screen Shot 2019-05-17 at 9 29 42 AM" src="https://user-images.githubusercontent.com/35148959/57934952-53e55500-7886-11e9-8186-b733851aa1a7.png">

Reply:
@mcoker I do not think we want padding here. It looks like a bug and it is not as clear that there is overflow.

Reply:
@christiemolloy opened https://github.com/patternfly/patternfly-next/issues/1839

Reply:
Sounds good thanks @mcoker